app/javascript/utilities/textAreaUtils.js

```diff
@@ -104,7 +104,8 @@ const getIndexOfCurrentWordAutocompleteSymbol = (content, selectionIndex) => {
   if (
     selectionIndex !== 0 &&
     previousCharacter !== ' ' &&
-    previousCharacter !== ''
+    previousCharacter !== '' &&
+    previousCharacter !== '\n'
```

Comment:
Refactor (non-blocking)

```suggestion
    ![' ', '', '\n'].includes(previousCharacter)
```

Reply:
oh nice - that's better!